src/jest.config.js

```diff
@@ -20,7 +20,8 @@ export default {
   moduleNameMapper: {
     // Module Name settings required to make chalk work with jest
     '#(.*)': '<rootDir>/node_modules/$1',
-    '^js-slang/context': '<rootDir>/__mocks__/context.ts'
+    '^js-slang/context': '<rootDir>/__mocks__/context.ts',
+    '^three.+.js': '<rootDir>/__mocks__/emptyModule.js',
```

Comment:
This is a fix to allow jest to understand the untranspiled Three examples like OrbitalControl and GLTFLoader

https://stackoverflow.com/questions/58363166/errors-cannot-use-import-statement-outside-a-module-three-orbitcontrol